apps/server/src/modules/task/task.module.ts

```diff
@@ -2,10 +2,11 @@ import { CopyHelperModule } from '@modules/copy-helper';
 import { FilesStorageClientModule } from '@modules/files-storage-client';
 import { Module } from '@nestjs/common';
 import { CourseRepo, SubmissionRepo, TaskRepo } from '@shared/repo';
+import { UserModule } from '@modules/user';
```

Comment:
TaskModule does not need UserModule, can it be removed from here?

Reply:
deleted